packages/nuxt-locokit/package.json

```diff
@@ -34,38 +34,38 @@
     "@locokit/designsystem": "*",
     "@locokit/i18n": "*",
     "@locokit/sdk": "*",
-    "@nuxt/kit": "^3.6.5",
+    "@nuxt/kit": "^3.7.1",
     "@pinia/nuxt": "^0.4.11",
     "@vee-validate/rules": "^4.11.3",
     "bootstrap-icons": "^1.10.5",
     "date-fns": "^2.30.0",
-    "pinia": "^2.1.3",
-    "primevue": "^3.32.1",
-    "tailwindcss": "^3.3.1",
+    "pinia": "^2.1.6",
+    "primevue": "^3.34.0",
+    "tailwindcss": "^3.3.3",
     "vee-validate": "^4.11.3",
-    "vue-i18n": "^9.3.0-beta.17"
+    "vue-i18n": "^9.3.0"
   },
   "devDependencies": {
-    "@nuxt/devtools": "^0.8.0",
-    "@nuxt/eslint-config": "^0.1.1",
-    "@nuxt/module-builder": "^0.4.0",
-    "@nuxt/schema": "^3.6.5",
-    "@nuxt/test-utils": "^3.6.5",
-    "@nuxtjs/eslint-config-typescript": "^12.0.0",
-    "@nuxtjs/tailwindcss": "6.7.0",
-    "@types/node": "^18.17.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.7",
-    "@typescript-eslint/parser": "^5.59.7",
+    "@nuxt/devtools": "^0.8.2",
+    "@nuxt/eslint-config": "^0.2.0",
+    "@nuxt/module-builder": "^0.5.1",
+    "@nuxt/schema": "^3.7.1",
+    "@nuxt/test-utils": "^3.7.1",
+    "@nuxtjs/eslint-config-typescript": "^12.1.0",
+    "@nuxtjs/tailwindcss": "6.8.0",
+    "@types/node": "18.17.0",
+    "@typescript-eslint/eslint-plugin": "5.59.7",
+    "@typescript-eslint/parser": "5.59.7",
     "@vue/eslint-config-typescript": "^11.0.3",
-    "autoprefixer": "^10.4.14",
-    "changelogen": "^0.5.4",
+    "autoprefixer": "^10.4.15",
+    "changelogen": "^0.5.5",
     "eslint": "^8.45.0",
     "eslint-plugin-nuxt": "^4.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-vue": "^9.14.0",
-    "nuxt": "^3.6.5",
+    "nuxt": "^3.7.1",
     "postcss": "^8.4.23",
-    "prettier": "^2.8.8",
+    "prettier": "2.8.8",
```

Comment:
The fact you fix the version of `prettier` and `@typescript-eslint/parser` / `@typescript-eslint/eslint-plugin` is to avoid issues with latest version ?

Reply:
It was to restore my local